README.md

```diff
@@ -90,13 +82,13 @@ sudo apt-get install postgresql postgresql-contrib
 Clone this repository to your desired folder:
 
 cd my-folder
-git clone git@github.com:David63011/vet-clinic-database.git
+git clone github.com/david63011/school-library.git
 
 ### Install
 
 Install this project with:
 
-cd vet-clinic-database
+cd oop
 
 ### Usage
 
```

Comment:
- [ ] KIndly fix the link to your MIT LICENSE. Currently, it displays a **_404 error page_**. It would be nice to fix this as it would make your README.md file look very professional. 👍

Reply:
fixed